### ISSUE TITLE
(installing brew) Failed during: /home/linuxbrew/.linuxbrew/bin/brew update --force

### ISSUE THREAD

**Issue by BrianJMRogers** [Linuxbrew/homebrew-core#9510]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [ ] have a problem with `brew install` (or `upgrade`, `reinstall`) a single formula? If it's a general `brew` problem please file this issue at Linuxbrew/brew: https://github.com/Linuxbrew/brew/issues/new/choose. If it's a tap (e.g. Brewsci/homebrew-bio) problem please file this issue at the tap.
- [ ] ran `brew update` and can still reproduce the problem?
- [ ] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
- What happened (include command output)
- What you expected to happen
- Step-by-step reproduction instructions (by running `brew install` commands)

Hello! None of the above seem to be my issue. I am running

 ```sh -c "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install.sh)" ``` 

and eventually crash at 
```HEAD is now at 1699a7ec1 cmd/vendor-install: Use HOMEBREW_BOTTLE_DOMAIN (#847)
Traceback (most recent call last):
        7: from /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/brew.rb:15:in `<main>'
        6: from /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/brew.rb:15:in `require_relative'
        5: from /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/global.rb:15:in `<top (required)>'
        4: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        3: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        2: from /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/system_command.rb:3:in `<top (required)>'
        1: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require': cannot load such file -- plist (LoadError)
Failed during: /home/linuxbrew/.linuxbrew/bin/brew update --force
```

I've tried the command multiple times. I have rm rf'd my /home/linuxbrew directory. My PATH is ```/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin```

I'm running this on Kubuntu.

**Comment by kingdavidmartins** [comment]
I get this exact same message. After setting brew up. I ran `brew install node` and got this exact same message. Now I get it for every command including brew doctor, config, gist-logs, etc. For some reason all formulas/commands output the following 

```
Traceback (most recent call last):
        7: from /home/kingdavidmartins/.linuxbrew/Library/Homebrew/brew.rb:15:in `<main>'
        6: from /home/kingdavidmartins/.linuxbrew/Library/Homebrew/brew.rb:15:in `require_relative'
        5: from /home/kingdavidmartins/.linuxbrew/Library/Homebrew/global.rb:15:in `<top (required)>'
        4: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        3: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        2: from /home/kingdavidmartins/.linuxbrew/Library/Homebrew/system_command.rb:3:in `<top (required)>'
        1: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require': cannot load such file -- plist (LoadError)
```

**Comment by iMichka** [comment]
This has been fixed with brew 1.7.6, see https://github.com/Linuxbrew/brew/issues/849 for more details and some workarounds if you still have the issue.
